theano/sandbox/cuda/tests/test_opt.py

```diff
@@ -153,7 +153,20 @@ def test_gpualloc():
     l = f.maker.fgraph.toposort()
     assert numpy.any([isinstance(x.op, cuda.GpuAlloc) for x in l])
 
+def test_gpuallocempty():
```

Comment:
A second blank line needs to go between this function declaration and the previous code block.